readme.md

```diff
@@ -941,6 +941,7 @@
 - [WebXR](https://github.com/msub2/awesome-webxr#readme) - Enables immersive virtual reality and augmented reality content on the web.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects to inspire and help bootstrap new documentation projects.
```

Comment:
The title here is not consistent with the readme title in https://github.com/readthedocs-examples/awesome-read-the-docs#readme It includes the word "Projects"

Reply:
Ay! Thanks! I had a blind spot there - it's best to call it "Awesome Read the Docs"... being a "project" seems very obvious here, everything in the list is a "project" :)